### PR TITLE
* fix for https://github.com/opbeat/opbeat_python/issues/95

### PR DESCRIPTION
**Pull request by jalvz**
* added test for requests

### CODE REVIEW

File: tests/instrumentation/requests_tests.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+class InstrumentRequestsTest(TestCase):
+    def setUp(self):
+        self.client = get_tempstoreclient()
+        opbeat.instrumentation.control.instrument()
+
+    def test_requests_instrumentation(self):
+        self.client.begin_transaction("transaction.test")
+        with trace("test_pipeline", "test"):
+            try:
+                requests.get('http://example.com')
```
Comment (piquadrat) [inline tests/instrumentation/requests_tests.py]: is there any way we can mock this? If at all possible, the tests shouldn't hit outside services.

File: setup.py [hunk 2]
```diff
@@ -71,6 +71,7 @@ def build_extension(self, ext):
     'webob',
     'pytz',
     'redis',
+    'requests',
```
Comment (piquadrat) [inline setup.py]: can you add this to test_requirements/requirements-base.txt as well? this ensures that it is added to the pip cache on travis.

File: tests/instrumentation/requests_tests.py (hunk 1 above)
Comment (jalvz) [inline tests/instrumentation/requests_tests.py]: I understand that as a general rule, but what is the problem with this case? I don't care what is the result or which exception raises (i could catch a broader exception type...)

The thing with mocks is that i am not sure if/how mock instrumentation can interfere with Opbeat instrumentation and silently invalidate the result. I rather prefer to test the real thing...

File: setup.py (hunk 2 above)
Comment (jalvz) [inline setup.py]: :+1:

**Pull request closed without merging**